x-pack/test/apm_api_integration/tests/diagnostics/index_templates.spec.ts

```diff
@@ -20,7 +20,7 @@ export default function ApiTest({ getService }: FtrProviderContext) {
   const start = new Date('2021-01-01T00:00:00.000Z').getTime();
   const end = new Date('2021-01-01T00:15:00.000Z').getTime() - 1;
 
-  registry.when('Diagnostics: Index Templates', { config: 'basic', archives: [] }, () => {
+  registry.when.skip('Diagnostics: Index Templates', { config: 'basic', archives: [] }, () => {
```

Comment:
Is this test skipped related with the other PR?

Reply:
Yes, I'm waiting for the [PR](https://github.com/elastic/kibana/pull/186870) to be merged and update this PR.